kulkunen/admin.py

```diff
@@ -1,28 +1,104 @@
 from django.contrib import admin
-from django_admin_json_editor.admin import JSONEditorWidget
 
-from .models import AccessControlResource, AccessControlSystem
+from .models import (
+    AccessControlGrant,
+    AccessControlResource,
+    AccessControlSystem,
+    AccessControlUser,
+)
 
 
 @admin.register(AccessControlSystem)
 class AccessControlSystemAdmin(admin.ModelAdmin):
-    def get_form(self, request, obj=None, **kwargs):
-        schema = {}
-        if obj is not None:
-            schema = obj.get_system_config_schema()
-        widget = JSONEditorWidget(schema, collapsed=False)
-        form = super().get_form(request, obj, widgets={'driver_config': widget}, **kwargs)
-        return form
+    pass
 
 
 @admin.register(AccessControlResource)
 class AccessControlResourceAdmin(admin.ModelAdmin):
-    list_display = ('resource', 'system', 'driver_identifier', 'active_grant_count')
-
-    def get_form(self, request, obj=None, **kwargs):
-        schema = {}
```

Comment:
Removed the JSONEditorWidget as it was not working properly and made it hard to edit the JSON in the Django admin.